pkg/sensors/exec/exec.go

```diff
@@ -75,21 +75,22 @@ func execParse(reader *bytes.Reader) (processapi.MsgProcess, bool, error) {
 	proc.Flags = exec.Flags
 	proc.Ktime = exec.Ktime
 	proc.AUID = exec.AUID
+	set_dummy_proc := func() {
+		proc.Size = processapi.MSG_SIZEOF_EXECVE
+		proc.Args = "enomem enomem"
+		proc.Filename = "enomem"
```

Comment:
Nice catch! thanks!

I think this is a bit "enomem" is somewhat confusing to users, so it might be a good opportunity to fix this as well.

How about something like:

```
	set_dummy_proc := func() {
		proc.Size = processapi.MSG_SIZEOF_EXECVE
		proc.Args = ""
		proc.Filename = ""
                proc.Flags = "parse-error"
```

@jrfastab WDT?

Reply:
Agree people usually ask what does enomem mean when they see it. So better names would be great. I think we can merge this PR and do that as a follow up though?